packages/date-picker/src/CalendarNav.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import styles from "./DatePicker.scss"
+
+export type CalendarNavProps = {
+  onPreviousClick: () => void
+  onNextClick: () => void
+}
+
+export const CalendarNav: React.VFC<CalendarNavProps> = ({
+  onPreviousClick,
+  onNextClick,
+}) => (
+  <div className={styles.navbar}>
+    <button
+      className={styles.arrows}
+      onClick={() => onPreviousClick()}
+      type="button"
+      aria-label="Previous Month"
```

Comment:
NB: This means that the component won't meet the `Internationalized` health metric. Not an issue at the moment as we don't have a solution for i18n in Kaizen right now. Something to keep in the back of your mind for the future.

Reply:
Yep that's on the roadmap for the next iteration when we do the DateRange! Thanks for flagging though.